test/suites/optional/multus.robot

```diff
@@ -0,0 +1,151 @@
+*** Settings ***
+Documentation       Tests for Multus and Bridge plugin on MicroShift
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/microshift-process.resource
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Variables ***
+${BRIDGE_INTERFACE}                 br-test
+${BRIDGE_NAD_YAML}                  ./assets/multus/bridge-nad.yaml
+${BRIDGE_POD_YAML}                  ./assets/multus/bridge-pod.yaml
+${BRIDGE_POD_NAME}                  test-bridge
+${BRIDGE_IP}                        10.10.0.10/24
+
+${PREEXISTING_BRIDGE_INTERFACE}     br-preexisting
+${PREEXISTING_BRIDGE_NAD_YAML}      ./assets/multus/bridge-preexisting-nad.yaml
+${PREEXISTING_BRIDGE_POD_YAML}      ./assets/multus/bridge-preexisting-pod.yaml
+${PREEXISTING_BRIDGE_POD_NAME}      test-bridge-preexisting
+${PREEXISTING_BRIDGE_IP}            10.10.1.10/24
+
+${NS}                               default
```

Comment:
Is there a special reason we're not using a dedicated namespace for this test?

Reply:
Not really, let me have a look

Reply:
done